package.json

```diff
@@ -42,6 +42,13 @@
     "url": "https://github.com/inigolabs/inigo-js/issues"
   },
   "devDependencies": {
-    "jest": "^29.6.4"
+    "jest": "^29.6.4",
+    "inigo-linux-amd64": "latest",
+    "inigo-linux-arm64": "latest",
+    "inigo-darwin-amd64": "latest",
+    "inigo-darwin-arm64": "latest",
+    "inigo-windows-amd64": "latest",
+    "inigo-windows-arm64": "latest",
```

Comment:
By the looks of it, these are not dev dependencies but actual dependencies

Reply:
I removed it.